Platformer2D/Platforms/Android/Properties/AndroidManifest.xml

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<manifest xmlns:android="http://schemas.android.com/apk/res/android">
-	<uses-sdk android:minSdkVersion="12" android:targetSdkVersion="12" />
-	<application></application>
-</manifest>
\ No newline at end of file
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="net.monogame.samples.platformer2d">
+	<uses-sdk android:minSdkVersion="21" android:targetSdkVersion="28" />
+	<application android:label="MonoGame.Samples.Platformer2D.Android"></application>
+        <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
```

Comment:
Why would we need Network Access permissions for a sample that is not using networking. 

Also, why increase the API version requirements, which limits the devices this sample can be deployed on

Reply:
I believe both of these changes are some kind of default coming in from an updated android SDK. I'll try to fix both of these.

Reply:
This is fixed now. I tested clean build and ran it on the emulator (Pixel XL, apiversion=29).